keyboards/ergodox_ez/config.h

```diff
@@ -80,8 +80,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define RGBW 1
 
-/* Set 0 if debouncing isn't needed */
-#define DEBOUNCE    15
+/* "debounce" is measured in keyboard scans. Some users reported
+ * needing values as high as 15, which was at the time around 50ms.
+ * If you don't define it here, the matrix code will default to
+ * 5, which is now closer to 10ms, but still plenty according to
+ * manufacturer specs.
+ */
+//#define DEBOUNCE    15
```

Comment:
I am not sure I want to drop debounce from 15 for all users. Concerned about potential support impact. Let's leave it 15 for now, and I will open it up via the configurator for people to experiment with.